paasta_tools/cli/cmds/validate.py

```diff
@@ -149,6 +165,31 @@ def get_service_path(service, soa_dir):
             return None
 
 
+def path_to_soa_dir_service(service_path):
+    soa_dir = os.path.dirname(service_path)
+    service = os.path.basename(service_path)
+
+    return soa_dir, service
+
+
+def validate_chronos(service_path):
+    soa_dir, service = path_to_soa_dir_service(service_path)
+    instance_type = 'chronos'
+
+    for cluster in list_clusters(service, soa_dir, instance_type):
```

Comment:
Yes, it does. I need to iterate over all of the clusters for `cjc = load_chronos_job_config(service, instance, cluster, False, soa_dir)`. I can try and clean up the code a bit to make this more clear.